src/Microsoft.AspNetCore.SignalR.Common/Internal/Protocol/PongMessage.cs

```diff
@@ -0,0 +1,12 @@
+namespace Microsoft.AspNetCore.SignalR.Internal.Protocol
+{
+    public class PongMessage:HubMessage
```

Comment:
I don't miss your awful formatting @anurse

Reply:
I can happily stop writing code for you if you'd prefer ;)